crypto/jwx.go

```diff
@@ -271,6 +240,29 @@ func signJWS(payload []byte, protectedHeaders map[string]interface{}, privateKey
 	return string(data), nil
 }
 
+func (client *Crypto) getPrivateKey(key interface{}) (crypto.Signer, string, error) {
```

Comment:
The `interface{}` type in the interface and this method are a bit smelly. But a nice solution doesn't come to mind.

Reply:
I agree, but I couldn't find another solution either...